torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -101,7 +101,9 @@ def runtime_wrapper(*args):
         num_intermediate_bases = runtime_metadata.num_intermediate_bases
 
         if keep_input_mutations and trace_joint:
-            num_graph_handled = runtime_metadata.num_mutated_graph_handled_indices
```

Comment:
nit: maybe just rename to `num_input_mutations_handled_by_autograd`, since we now have this (very confusing) distinction worth highlighting:

- input mutations that are kept outside the graph
- input mutations that are kept in the graph, and are fully hidden from autograd (triton kernels)
- input mutation that are kept in the graph, but that autograd must know about so we increment their version counters (the var in question here)